appserver/tests/payara-samples/classloader-data-api/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>fish.payara.samples</groupId>
+        <artifactId>payara-samples</artifactId>
+        <version>5.2020.8-SNAPSHOT</version>
+    </parent>
+    
+    <groupId>fish.payara.samples</groupId>
+    <artifactId>classloader-data-api</artifactId>
+    <version>5.2020.8-SNAPSHOT</version>
+    
+    <packaging>war</packaging>
+    <name>Payara Samples - ClassLoader Data API</name>
+    <description>A REST API Application for retrieving an instance count of 
+        WebappClassLoader from an instance of Payara Server</description>
```

Comment:
Need the same functionality for ASURLClassLoader (parent of EARClassLoader EJBClassLoader etc)